src/OpenLoco/Title.cpp

```diff
@@ -81,13 +161,73 @@ namespace OpenLoco::Title
         initialiseViewports();
         sub_4284C8();
         Gui::init();
-        sub_444357();
+        reset();
         Gfx::invalidateScreen();
         resetScreenAge();
 
         Audio::playTitleScreenMusic();
     }
 
+    // 0x00444387
+    void update()
+    {
+        if (!isTitleMode())
+            return;
+
+        resetScreenAge();
+
+        if (_waitCounter > 0)
+        {
+            _waitCounter -= 1;
+            return;
+        }
+
+        do
+        {
+            auto& command = *_sequenceIterator++;
+            std::visit(overloaded{
+                           [](WaitStep step) {
+                               _waitCounter = step.duration / 4;
+                           },
+                           [](ReloadStep step) {
+                               loadTitle();
+                               Gfx::invalidateScreen();
+                               resetScreenAge();
+                               addr<0x50C19A, uint16_t>() = 55000;
+                           },
+                           [](MoveStep step) {
+                               if (addr<0x00525E28, uint32_t>() & 1)
+                               {
+                                   auto pos = Map::map_pos(step) + Map::map_pos(16, 16);
+                                   auto height = Map::TileManager::getHeight(pos);
+                                   auto main = Ui::WindowManager::getMainWindow();
+                                   if (main != nullptr)
+                                   {
+                                       auto pos3d = Map::map_pos3(pos, height.landHeight);
```

Comment:
This no longer compiles since #809.

Reply:
Yeah I didn't want to support slicing, pass ```pos.x, pos.y``` instead.